platform/features/layout/src/LayoutController.js

```diff
@@ -79,6 +83,11 @@ define(
                     ],
                     dimensions: self.defaultDimensions()
                 };
+
+                // Store the id so that the newly-dropped object
+                // gets selected during refresh composition
+                self.droppedFrameId = id;
```

Comment:
I think it'd be better to rename the variable to droppedId and keep the comment. That way the variable is not tied to just one method/action and is more generic that can be used elsewhere if needed.